funcs/func_frame_drop.c

```diff
@@ -234,8 +238,10 @@ static int frame_drop_helper(struct ast_channel *chan, const char *cmd, char *da
 		framedata->list_type = TX;
 	}
 
-	buffer = ast_malloc(sizeof(value) + 3); /* leading and trailing comma and null terminator */
-	snprintf(buffer, sizeof(value) + 2, ",%s,", value);
+	if (ast_asprintf(&buffer, ",%s,", value) < 0) { /* leading and trailing comma and null terminator */
```

Comment:
The comment doesn’t make sense anymore, at least the part about the “null terminator.”